### PR TITLE
algorithm: reverse

### PR DESCRIPTION
**Pull request by SczSca**
### Describe your change:

- [X] Add an algorithm: Invert Array.
Algorithm to invert elements in an array: 
[a,b,c,d] //output [d,c,b,a]

### Checklist:

- [ X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [X ] This pull request is all my own work -- I have not plagiarized.
- [ X] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [X ] All new JavaScript files are placed inside an existing directory.
- [ X] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [ X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Data-Structures/Array/InvertArray.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* https://www.geeksforgeeks.org/write-a-program-to-reverse-an-array-or-string/ here explains ways to solve this problem in other languages*/
+const invertArr = (arr) => {
+    let temp = 0;
+    for(let i = 0, j = arr.length - 1; i < arr.length/2; i++, j--){ //LIMIT SPECIFIES THE AMOUNT OF INVERT ACTIONS
+        temp = arr[i];
+        arr[i] = arr[j];
+        arr[j] = temp;
+
```
Comment (syedjafer) [inline Data-Structures/Array/InvertArray.js:8]: Please user standard js for code style correction. 
Refer: Contributing.md - https://github.com/TheAlgorithms/JavaScript/blob/master/CONTRIBUTING.md#coding-style

For Standard JS - https://standardjs.com/

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please...
1. Name this `reverse`
2. Use a proper JSDoc comment
3. Why is your comment screaming at me?
4. Don't use a `const Array` in your tests (shadows the built-in `Array`) - just inline the arrays
5. Consider using `each` here
6. Declare `temp` as `const` inside the loop

**Comment by SczSca** [comment]
> Please...
> 
> 1. Name this `reverse`
> 2. Use a proper JSDoc comment
> 3. Why is your comment screaming at me?
> 4. Don't use a `const Array` in your tests (shadows the built-in `Array`) - just inline the arrays
> 5. Consider using `each` here
> 6. Declare `temp` as `const` inside the loop

Just to be sure:
1. You mean changing name files to 'Reverse.js' and 'Reverse.test.js' ?
2/3. Sorry I am still learning to being consistent
5. For each wouldn't work because it loops over half array

Sorry for these mistakes, I'm a beginner student that heard and is learning about the open-source software

**Comment by appgurueu** [comment]
>     1. You mean changing name files to 'Reverse.js' and 'Reverse.test.js' ?
>        2/3. Sorry I am still learning to being consistent

Not only the file names, also the function name.

>     2. For each wouldn't work because it loops over half array

Sorry, I should've made myself more clear. I did not mean "use for-each for reversing". I meant use Jest's `each` for testing.

**Comment by SczSca** [comment]
> > ```
> > 1. You mean changing name files to 'Reverse.js' and 'Reverse.test.js' ?
> >    2/3. Sorry I am still learning to being consistent
> > ```
> 
> Not only the file names, also the function name.
> 
> > ```
> > 2. For each wouldn't work because it loops over half array
> > ```
> 
> Sorry, I should've made myself more clear. I did not mean "use for-each for reversing". I meant use Jest's `each` for testing.

Wanted to sent the commit to my fork to change the name files, but sent it here anyways...

**Comment by SczSca** [comment]
> > ```
> > 1. You mean changing name files to 'Reverse.js' and 'Reverse.test.js' ?
> >    2/3. Sorry I am still learning to being consistent
> > ```
> 
> Not only the file names, also the function name.
> 
> > ```
> > 2. For each wouldn't work because it loops over half array
> > ```
> 
> Sorry, I should've made myself more clear. I did not mean "use for-each for reversing". I meant use Jest's `each` for testing.

Everything is done, I guess.
Sorry for the inconveniences and the amount of commits made. I'm a beginner student that heard a talk about open-source software in Hacktoberfest and wanted to give it a try with some code that was made in one of my classes

**Review by appgurueu — COMMENTED** [review]
You forgot to rename the files...

**Comment by SczSca** [comment]
> You forgot to rename the files...

the renamed file names are not reflected on the last two commits? I see te files renamed, but I might be wrong

image

image

image

**Review by appgurueu — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**